.circleci/ci-packages.txt

```diff
@@ -137,7 +137,7 @@ perl-Module-Implementation-0.09
 perl-Module-Pluggable-5.2
 perl-Module-Runtime-0.016
 perl-Module-Runtime-Conflicts-0.003
-perl-Mojo-IOLoop-ReadWriteProcess-0.25
+perl-Mojo-IOLoop-ReadWriteProcess

```

Comment:
This is only to temporarily fix the cache creation. The next nightly job will insert the (updated) version here.